### PR TITLE
Fixes #1235: CustomTreeCtrl edit label remains stuck forever

### LINKED ISSUE #1235: CustomTreeCtrl edit label remains stuck forever sometimes
Windows 10 (or Windows 7)
wxPython 4.0.6 (pip)
Python 3.7.3 [32bit]

From what I can tell, it seems that I'm able to somehow select the label from behind the TextCtrl and trigger a second edit label instance that can't be closed.
Or maybe KillFocus doesn't trigger correctly? I dunno.

Here's me reproducing it with the example from wxDemo:

Another example (from another GUI):

### PR DESCRIPTION
**Pull request by Nyral**
I added a condition to check that an edit control doesn't already exist.
For more info you can read [cbeytas's explanation](https://github.com/wxWidgets/Phoenix/issues/1235#issuecomment-496777121) about the issue.

### CODE REVIEW

File: wx/lib/agw/customtreectrl.py [hunk 1]
```diff
@@ -7733,8 +7733,9 @@ def Edit(self, item):
         if self._editCtrl != None and item != self._editCtrl.item():
             self._editCtrl.StopEditing()
 
-        self._editCtrl = TreeTextCtrl(self, item=item)
-        self._editCtrl.SetFocus()
+        if self._editCtrl == None:
```
Comment (RobinD42) [inline wx/lib/agw/customtreectrl.py]: using `is` in place of `==` would probably be better

Comment (Nyral) [reply wx/lib/agw/customtreectrl.py]: That's how None comparisons are handled everywhere else in file.

Comment (Nyral) [reply wx/lib/agw/customtreectrl.py]: Done.

**Comment by Nyral** [comment]
@RobinD42 Done.

**Comment by Nyral** [comment]
Thanks!

**Pull request merged**